aws-devicefarm-gradle-plugin/src/main/groovy/com.amazonaws.devicefarm/extension/DeviceFarmExtension.groovy

```diff
@@ -42,6 +46,24 @@ class DeviceFarmExtension {
      */
     String endpointOverride
 
+    /**
+     * [Optional] Wait for DeviceFarm to finish running the tests and consume results
+     * Default: False
+     */
+    boolean wait = false
+
+    /**
+     * [Optional] Produce output for Test Run results in format
+     * Default: JUnit
+     */
+    OutputType outputType = DeviceFarmExtension.OutputType.JUnit
```

Comment:
Does having this value as the default break backwards compatibility?

Reply:
As this is only engaged when `useWait()` is called in the gradle closure I don't think there's anything that'd break with this.